tutorials/welding_example_02_weaving.ipynb

```diff
@@ -75,10 +75,10 @@
    "source": [
     "groove = get_groove(\n",
     "    groove_type=\"VGroove\",\n",
-    "    workpiece_thickness=Q_(0.5, \"cm\"),\n",
-    "    groove_angle=Q_(50, \"deg\"),\n",
-    "    root_face=Q_(1, \"mm\"),\n",
-    "    root_gap=Q_(1, \"mm\"),\n",
+    "    workpiece_thickness=\"0.5 cm\",\n",
```

Comment:
Would it also have been possible to pass a tuple[Union[float, int], str], e.g. (1, "mm")? I would prefer such an option, if these quantities are actually the result of a computation.

Reply:
not sure at this specific point since I didn't touch the internals of the function. Just fixed an issue that popped up due to the function using the LCS. Anyways, all newer functions that support units should usually provide several format options for quantities. I think a tuple is also possible, but I have to check it.